primer.md

```diff
@@ -196,7 +196,7 @@ elements that form a layered architecture model.
    with any HTTP method and with request and response messages.
 
 If required to assure broader interoperability, the CloudEvents specification
-set provides specific constraints for event delivery using a particular
+set, (may or should) provide specific constraints for event delivery using a particular
```

Comment:
If we change it, I'd prefer `...the CloudEvents specification set will provide specific...`
I'm not sure how to interpret the `(may or should)`